packer/scylla_install_image

```diff
@@ -211,6 +211,15 @@ WantedBy=multi-user.target
     if args.target_cloud == 'azure':
         with open('/etc/hosts', 'a') as f:
             f.write('\n\n169.254.169.254    metadata.azure.internal\n')
+        with open('/etc/ssh/sshd_config.d/50-cloudimg-settings.conf') as f:
+            cfg = f.read()
+        cfg = re.sub('ClientAliveInterval 120', 'ClientAliveInterval 180', cfg, flags=re.MULTILINE)
+        with open('/etc/ssh/sshd_config.d/50-cloudimg-settings.conf', 'w') as f:
+            f.write(cfg)
+        grub = re.sub(r'^GRUB_CMDLINE_LINUX="(.+)"$', r'GRUB_CMDLINE_LINUX="\1 net.ifnames=0 clocksource=tsc tsc=reliable rootdelay=300"', grub, flags=re.MULTILINE)
```

Comment:
Why it is copying exiting GRUB code defined at [line 164 - 169](https://github.com/scylladb/scylla-machine-image/blob/b5da657fa207d0c0069df993068a46531381a6cd/packer/scylla_install_image#L164)?
I think these codes should merge like swap setup (it has cloud specific variable `swap_opt`).

Reply:
Changed, please review @syuu1228